docs/Lets-Write-a-Test.md

```diff
@@ -18,6 +18,8 @@ We prefer to install PE once and then run a set of tests, so PE installation sho
     $ cd setup
     $ cat > install.rb << RUBY
     test_name "Installing Puppet Enterprise" do
+    # Particularly when installing a larger number of hosts, this can reduce the execution time.
+    options[:run_in_parallel] = ["configuration", "install"]
```

Comment:
The code instances below are using the value 'configure' for this.

Running these tests in parallel seems like a big enough thing that it could use it's own doc. Would you mind writing up a small one, so that you can highlight these two use cases (these coded values vs running something like `on()` with `:run_in_parallel` set to `true`)?  I think a question that will come up when people see this is whether or not these are all the values that can be had for this setting as well. Having its own doc where you clearly delineate these different use cases would be helpful, rather than having people need to poke around in various docs for the different ways.

You can also highlight the environment variable method for passing as well there.